src/app/persons/persons.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { QueryRef } from 'apollo-angular';
+import { Subscription } from 'rxjs';
+import { PersonWithAllTypeFragment, PersonsWithAllQuery, Exact, InputMaybe, PersonWhere, PersonsWithAllGQL } from '../generated/graphql';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
```

Comment:
This component is root component. No logic here, all must be moved to experience form.